Source/Scene/Expression.js

```diff
@@ -62,7 +62,16 @@ define([
         asin : Math.asin,
         atan : Math.atan,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        degrees : CesiumMath.toDegrees,
+        sign : Math.sign,
+        floor : Math.floor,
+        ceil : Math.ceil,
+        round : Math.round,
+        exp : Math.exp,
+        exp2 : CesiumMath.exp2,
```

Comment:
I don't think exp2 is essential to add to `CesiumMath`. Instead create a function in `Expression.js` for `exp2` and reference that in the `unaryFunctions` list.